Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7.3-alpine AS base
+FROM python:3.8.0b3-slim-buster AS base
```

Comment:
I think it's fair to keep it at the non-beta release, so `python:3.7-slim` suffices, looking at https://hub.docker.com/_/python.